state-chain/chains/src/btc.rs

```diff
@@ -954,7 +963,7 @@ impl<T: SerializeBtc> SerializeBtc for &[T] {
 #[derive(Encode, Decode, TypeInfo, MaxEncodedLen, Clone, RuntimeDebug, PartialEq, Eq)]
 pub enum BitcoinOp {
 	PushUint { value: u32 },
-	PushBytes { bytes: BoundedVec<u8, ConstU32<MAX_SEGWIT_PROGRAM_BYTES>> },
+	PushBytes { bytes: BoundedVec<u8, ConstU32<MAX_PUSHABLE_BYTES>> },
```

Comment:
This seems a bit unnecessary? `MAX_SEGWIT_PROGRAM_BYTES` was already more than enough.

Reply:
What do you mean? `MAX_SEGWIT_PROGRAM_BYTES` is only 40 bytes, but we want to allow up to 80 bytes for our use case. It also makes more sense. `MAX_SEGWIT_PROGRAM_BYTES` was only used because that happened to be what was being pushed, however bitcoin script allows pushing up to `MAX_PUSHABLE_BYTES`, so that's more correct.

Reply:
Sorry I thought I saw 128, must have misread.